man/fi_endpoint.3.md

```diff
@@ -472,6 +472,11 @@ The following option levels and option names and parameters are defined.
   exchange between peer endpoints using the fi_connect, fi_accept,
   and fi_reject operations.  This option is read only.
 
+  The available space in CM messages may differ between endpoints created from
+  different domains. The size returned reflects the properties of the domain
+  from which the endpoint was created, except in the case of passive endpoints,
+  in which the size reflects the maximum size across all domains.
```

Comment:
Also, does this separate paragraph render properly as the `<dd>` part of the definition for `FI_OPT_CM_DATA_SIZE` in both (nroff) man pages and the HTML rendering of them?